clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/TransferReadGroupV2Key.scala

```diff
@@ -0,0 +1,23 @@
+package org.broadinstitute.clio.transfer.model
+
+import enumeratum._
+import scala.collection.immutable.IndexedSeq
+
+sealed trait TransferReadGroupLocation extends EnumEntry
+
+object TransferReadGroupLocation extends Enum[TransferReadGroupLocation] {
```

Comment:
I think we'll eventually want to rename this to something more generic like `FileLocation` and use it for gvcfs etc. in addition to ubams. Your call on if you want to do it now or not.